agentops/llm_tracker.py

```diff
@@ -151,9 +155,9 @@ def handle_stream_chunk(chunk: ChatCompletionChunk):
                 self.llm_event.agent_id = check_call_stack_for_agent_id()
                 self.llm_event.model = chunk.model
                 self.llm_event.prompt = kwargs["messages"]
-                choice = chunk.choices[
-                    0
-                ]  # NOTE: We assume for completion only choices[0] is relevant
+
+                # NOTE: We assume for completion only choices[0] is relevant
+                choice = chunk.choices[0]
```

Comment:
We use `Black Formatter` to format our code. It does the weird multi-line thing for the `[0]`. I think it looks a bit weird, but it makes the code look better overall. Could you leave this alone or install Black Formatter, so that this line doesn't keep getting changed back and forth? Thanks!

Reply:
I have black formatter installed. I thought moving comment to a different line will be better.
I'll move it back if that's what you prefer.